src/actions/API.actions.js

```diff
@@ -407,12 +338,14 @@ export function pushCustomThemeToServer(customTheme){
     return;
   }
        url = BASE_URL+'/aaa/changeUserSettings.json?'
-          +'key=custom_theme_value&value='+customTheme
-          +'&access_token='+cookies.get('loggedIn');
+          +'key1=customThemeValue&value1='+customTheme
```

Comment:
It was implemented in https://github.com/fossasia/chat.susi.ai/pull/670 by @isuruAb . The custom theme call is made to server from the theme dialog and not from settings route. So sending it seperately